TheAmazingAudioEngine/Utilities/AEIOAudioUnit.m

```diff
@@ -48,6 +49,69 @@ @interface AEIOAudioUnit ()
 #endif
 @end
 
+
+////////////////////////////////////////////////////////////////////////////////
+// sets nr of seconds between reports, 0 = no reporting
+#define AE_REPORT_TIME 2
+
+#if AE_REPORT_TIME
+
+static OSStatus notifyCallback(
+	void *							inRefCon,
+	AudioUnitRenderActionFlags *	ioActionFlags,
+	const AudioTimeStamp *			inTimeStamp,
+	UInt32							inBusNumber,
+	UInt32							inNumberFrames,
+	AudioBufferList * __nullable	ioData)
+{
+	static UInt64 startTime = 0;
```

Comment:
Yes, that's how Michael had it initially, but eventually I decided statics, because it is easier to copy-paste the code. Having said that, for public code, it probably is better to encapsulate it in an object or struct, but I prefer it not to be part of the actual module.